content/open-source/slsa/what-is-slsa.md

```diff
@@ -0,0 +1,83 @@
+
+### Level 4 
+This level represents the highest level of trust possible in the SLSA framework and provides the strongest possible assurance that the software has not been tampered with. It is one of the most difficult levels to achieve, and may not be necessary (or even possible) for all software projects. 
+For this level, all changes to the codebase require a two-person review. The software must also be produced through a hermetic, reproducible build process. 
+
+A _hermetic_ build is defined by a build where all build steps, sources, and dependencies are declared with _immutable references_, or identifiers that are guaranteed to point to an immutable artifact whose integrity is authenticated by a cryptographic hash of its contents. These build steps, sources, and dependencies must also include their transitive counterparts, or dependencies of dependencies. The build steps must also run without network access. 
+
+A _reproducible_ build is one in which the build steps will always reproduce “bit-for-bit” identical output. If software cannot meet this requirement, a justification must be provided. Though the build service may not verify reproducibility, consumers should be able to check its reproducibility and can reject the build if they cannot reproduce the results.
+
+## SLSA Tools and Practices
+
+SLSA is a relatively new framework, with supporting tools and practices still actively evolving. Some of the requirements listed in the levels above can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [Github repository](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga), one of the hardest requirements, with Github Actions in February 2023.  
+
+To verify the SLSA provenance of a piece of software, you can use the [`slsa-verifier` tool](https://github.com/slsa-framework/slsa-verifier), which verifies provenances generated by the `slsa-generator` tool or Google Cloud Build. Other tools, like Chainguard’s [Enforce](https://www.chainguard.dev/chainguard-enforce) or Sigstore’s open source [Policy Controller](https://docs.sigstore.dev/policy-controller/overview/) allow you to create policies around SLSA requirements in your Kubernetes cluster. 
```

Comment:
"provenances" sounds weird, I would try to rephrase to keep that word singular

Reply:
done!